ios/sandwich/QonversionSandwich.swift

```diff
@@ -260,7 +262,7 @@ extension QonversionSandwich: Qonversion.PurchasesDelegate {
   public func qonversionDidReceiveUpdatedPermissions(_ permissions: [String : Qonversion.Permission]) {
     let permissionsDict: BridgeData = permissions.mapValues { $0.toMap() }
     
-    qonversionEventListener.qonversionDidReceiveUpdatedPermissions(permissionsDict)
+    qonversionEventListener?.qonversionDidReceiveUpdatedPermissions(permissionsDict as [String: Any])
```

Comment:
clearEmptyValues?

Reply:
No need, because we are sure that iOS native SDK couldn't return `nil` inside the Dictionary. It's impossible in `Obj-C`